packages/neon-domain/__integration__/urls.ts

```diff
@@ -0,0 +1,26 @@
+
+export function getMainnetUrls(): string[] {
```

Comment:
This file is failing the integration tests because it tries to find a test inside. Let's just add this function into `__integration__/NeonNS.ts` and delete this file.

Reply:
fixed, but maybe there is a way to have the urls.ts somewhere else to use it for all tests?

Reply:
yeah that can be an improvement in a separate PR.